platform/resources/user-retirement-collector.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# prepare tubular
+cd $WORKSPACE/tubular
+pip install -r requirements.txt
+
+# Create the directory where we will populate properties files, one per
+# downstream build.
+mkdir $LEARNERS_TO_RETIRE_PROPERTIES_DIR
+
+# Call the script to collect the list of learners that are to be retired.
+scripts/get_learners_to_retire.py
+    --secrets=$WORKSPACE/user-retirement-secure/$ENVIRONMENT.yml
+    --output_dir=$LEARNERS_TO_RETIRE_PROPERTIES_DIR
```

Comment:
@doctoryes this was the CLI I was predicting for get_learners_to_retire.py.  This doesn't actually cover HTTP basic auth, but I'm tempted to also squeeze that into the user-retirement-secure repo in the same yaml blobs containing the client_secret.

The file names for each properties file under `$LEARNERS_TO_RETIRE_PROPERTIES_DIR` isn't important, other than they don't collide.